src/getDynatraceEnv.ts

```diff
@@ -28,6 +30,10 @@ export function getDynatraceEnv(env: NodeJS.ProcessEnv = process.env): Dynatrace
     );
   }
 
+  if (isNaN(grailBudgetGB) || grailBudgetGB <= 0) {
```

Comment:
Changelog says

> Added Grail budget tracking with `DT_GRAIL_QUERY_BUDGET_GB` environment variable (default: 1000 GB, setting it to `-1` disables it).

Am I missing an early exit or way around this check in cases where a user sets it to `-1`?